### ISSUE TITLE
new permissions google chrome extension permissions

### ISSUE THREAD

**Issue by jplusc** [mrmrs/pesticide#65]
@paulmolluzzo  The chrome store version of this now want's new permissions.

it is complaining about **"read and change your data on wwww.googleapis.com"**  Is this just google being overly protective, or was there actually an update to the extension that needs new permissions?    I see this repo hasn't been updated in quite a few years, which makes the request for new permissions kinda suspicious. 

thanks

**Comment by paulmolluzzo** [comment]
This was an update I made to investigate options for future updates. I published a new update removing these permission requests before this issue was opened and it's awaiting review to be published in the store. it'll be in v1.6.

If you chose to deny those permissions Chrome should not update the extension and should continue to use the previously installed version instead.

> I see this repo hasn't been updated in quite a few years, which makes the request for new permissions kinda suspicious.

I don't own this repository and it seems generally abandoned. Updates made since version 1.2 are not in GitHub for that reason.
